rt/rs/microprofile-client/pom.xml

```diff
@@ -182,14 +182,18 @@
         </dependency>
         <dependency>
             <groupId>org.wiremock</groupId>
-            <artifactId>wiremock</artifactId>
+            <artifactId>wiremock-standalone</artifactId>
```

Comment:
Ah yeah, wiremock still uses Jetty 11

Reply:
Yes, actually this is the major reason why I hold this PR that long. This is the last piece that requires Jetty11, until I found other projects which face the similar issue using wiremock-standalone

Reply:
I will open a ticket (but yeah, `wiremock-standalone` is a way to go)